charts/kyverno/templates/NOTES.txt

```diff
@@ -21,11 +21,6 @@ The following components have been installed in your cluster:
 - Grafana dashboard
 {{- end }}
 
-{{ if not .Values.admissionController.replicas }}
-⚠️  WARNING: Setting the admission controller replica count below 3 means Kyverno is not running in high availability mode.
-{{- else if lt (int .Values.admissionController.replicas) 3 }}
-⚠️  WARNING: Setting the admission controller replica count below 3 means Kyverno is not running in high availability mode.
-{{- end }}
```

Comment:
Would it be possible to keep this but change `3` to `2`?  I think we still want to warn folks they are not in HA mode with 2+ replicas.

Reply:
Yes, that should be done.